packages/edit-site/src/components/navigation-sidebar/index.js

```diff
@@ -23,8 +23,12 @@ export default function NavigationSidebar() {
 	return (
 		<>
 			<NavigationToggle isOpen={ isNavigationOpen } />
-			<NavigationPanel isOpen={ isNavigationOpen } />
-			<NavigationPanelPreviewSlot />
+			{ isNavigationOpen && (
+				<>
+					<NavigationPanel isOpen={ isNavigationOpen } />
```

Comment:
If this fix works and is agreed upon, I suggest we follow-up in one of two ways:
* Update the NavigationPanel component above to always assume `isOpen`
* Fix the underlying bugs that prevent us from having this always mounted and then revert this to its previous state.

Reply:
I don't mind this as a temporary measure.
I think we should probably add a comment to explain why we do this.

> Update the NavigationPanel component above to always assume isOpen
maybe we can skip this and go straight to:
> Fix the underlying bugs that prevent us from having this always mounted and then revert this to its previous state.

---
I may have some ideas about the causes, the Rest API calls (actions) that cause the creation of the template parts on the server should invalidate the cache of the `getEntityRecords` calls. There's an API to invalidate cache in core-data but I'm not sure which API calls trigger the drafts to be created.

Reply:
> maybe we can skip this and go straight to:

Sounds good (if this is actually working 🤞 ), Noah saw a similar error in another test in CI below, but it could be the difference of a `waitForSelector`.

> but I'm not sure which API calls trigger the drafts to be created.

Too many it feels like.  But I assume this would be a conflict between the template part rest filter and the call to create auto-drafts on the `block_editor_settings` filter?